src/masonite/inertia/core/InertiaResponse.py

```diff
@@ -60,29 +60,26 @@ def render(self, component, props={}, custom_root_view=None):
 
     def location(self, url):
         # TODO: make request with 409 code and X-Inertia-Location: url header
-        self.request.header("X-Inertia-Location", url)
-        self.request.status(409)
-        # self.rendered_template = self.view("").rendered_template
-        # self.rendered_template = self.view(
-        #     custom_root_view if custom_root_view else self.root_view,
-        #     {"page": html.escape(json.dumps(page_data))},
-        # ).rendered_template
+        response = self.container.make("Response")
```

Comment:
Here I have done that but that's not correct, I am struggling a bit with the different concepts View, Responsable and the Response object.
The feature at stake here is `location` that we can use with inertia to make a inertia view redirect to an external link. You can find the implementation in PHP in this one line here: 
https://github.com/inertiajs/inertia-laravel/blob/master/src/ResponseFactory.php#L76

We need to construct a response with a code 409 and a specific header, but I don't know how to return it

Reply:
```python
response.header('X-Inertia-Location', 'url here')
response.status(409)
```

You might also need to set the actual string response so if above doesn't work you can do:

```python
response.header('X-Inertia-Location', 'url here')
response.make('', status=409)
```

Reply:
It is a little confusing actually getting the response class. You have to resolve the actual class like this:

```python
from masonite.response import Response
response = self.container.make(Response)
```

notice its the class and not a string you are making

Reply:
I am getting this error when setting header : 
```
>       response.header("X-Inertia-Location", url)
E       AttributeError: 'Response' object has no attribute 'header'
```

I verified that I got the latest version of masonite though, that's weird ...

Reply:
latest is on the develop branch not the 3.0 branch I think

Reply:
yes I know : I am running "pip install git+https://github.com/MasoniteFramework/masonite@develop" but apparently the version does not installed correctly because yeah it looks like this is not the latest masonite version